lib/spack/spack/test/cmd/ci.py

```diff
@@ -1652,6 +1652,7 @@ def fake_dyn_mapping_urlopener(*args, **kwargs):
                 assert "unallowed_field" not in job
 
 
+@pytest.mark.parametrize("affected_packages", [None, "trivial-pkg-with-valid-hash"])
```

Comment:
```suggestion
@pytest.mark.parametrize("affected_packages", [[], ["trivial-pkg-with-valid-hash"]])
```

Reply:
I had this, but didn't like how it looked in the test name.